content/blog/2016/2016-07-26-blueocean-alpha-4.adoc

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Jenkins Blue Ocean 1.0-alpha-4 released"
+tags:
+- blueocean
+- ux
+- pipeline
+author: i386
+---
+
+We've released Blue Ocean 1.0-alpha-4 to the experimental update center.
```

Comment:
its really hard to keep track as so much has changed. Perhaps this could be trimmed to a quick note? or just not worth doing? 

More just a heartbeat of "things are still happening... fast".